packages/utils/.gitignore

```diff
@@ -0,0 +1 @@
+lib
```

Comment:
no reason to have `.gitignore` in packages, we only duplicating same ignore all over packages, please add `lib` to root `gitignore` and remove this file

Reply:
:+1: